### PR TITLE
Update CONTRIBUTING documentation with CIM dev setup

### PR DESCRIPTION
**Pull request by jtomasek**

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -91,9 +91,52 @@ apply for `libs/locales` as well.
...
+   ```
+2. In one terminal run `libs/ui-lib` in watch mode.  
+   The script builds and publishes the `@openshift-assisted/ui-lib` package to the local registry
+   everytime you make a change.
+
+   ```bash
+   yarn workspace @openshift-assisted/ui-lib watch
+   ```
+
+3. In one terminal run `libs/locales` in watch mode.
```
Comment (jkilzi) [inline docs/CONTRIBUTING.md:121]:
```suggestion
3. In another terminal run `libs/locales` in watch mode.  
```

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -91,9 +91,52 @@ apply for `libs/locales` as well.
...
+4. Follow the stolostron/console
+   [development setup guide](https://github.com/stolostron/console#running).
+5. To use locally published @openshift-assisted packages inside the stolostron/console run the
+   following commands
+   ```bash
+   cd frontend
+   yalc link @openshift-assisted/ui-lib
+   yalc link @openshift-assisted/locales
+   ```
+   Note that these commands need to be re-executed every time the `npm install` is run in `frontend`
```
Comment (jkilzi) [inline docs/CONTRIBUTING.md:138]:
```suggestion
   Note that these commands need to be re-executed every time the `npm install` is run in the `frontend`
```

**Review by jkilzi — APPROVED** [review]

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (jtomasek) [reply docs/CONTRIBUTING.md]: done

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (jtomasek) [reply docs/CONTRIBUTING.md]: done, thanks!

**Review by celdrake — APPROVED** [review]

**Review by rawagner — APPROVED** [review]

**Pull request merged**